### PR TITLE
v6.19.5

### PR DESCRIPTION
**Pull request by MBilalShafi**
### Release the packages

- [ ] Checkout the last version of the working branch
- [ ] Make sure you have the latest dependencies installed: `yarn`.
- [ ] Build the packages: `yarn release:build`.
- [ ] Release the versions on npm: `yarn release:publish` (you need your 2FA device).
- [ ] Push the newly created tag: `yarn release:tag`.

### Publish the documentation

The documentation must be updated on the `docs-vX` branch (`docs-v4` for `v4.X` releases, `docs-v5` for `v5.X` releases, ...)

- [ ] Push the working branch on the documentation release branch to deploy the documentation with the latest changes.

```bash
git push upstream master:docs-v6 -f
```

You can follow the deployment process [on the Netlify Dashboard](https://app.netlify.com/sites/material-ui-x/deploys?filter=docs-v6)
Once deployed, it will be accessible at https://material-ui-x.netlify.app/ for the `docs-v6` deployment.

### Publish GitHub release

- [ ] After documentation is deployed, publish a new release on [GitHub releases page](https://github.com/mui/mui-x/releases)

### CODE REVIEW

**Review by cherniavskii — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,66 @@
...
+
+### Docs
+
+- [docs] Clarify Pickers `Component composition` section (#12147) @LukasTy
+- [docs] Fix 301 redirection to StackBlitz @oliviertassinari
+- [docs] Fix 301 to Material UI @oliviertassinari
+- [docs] Fix 301 to Material UI @oliviertassinari
+- [docs] Fix 404 links to translation source @oliviertassinari
+- [docs] Fix dead link to translations @oliviertassinari
+- [docs] Fix typo for AdapterDateFnsV3 (#12037) @flaviendelangle
```
Comment (flaviendelangle) [inline CHANGELOG.md:52]:
```suggestion
- [docs] Fix typo for `AdapterDateFnsV3` (#12037) @flaviendelangle
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,66 @@
...
+- [docs] Improve performance on charts entry point @oliviertassinari
```
Comment (flaviendelangle) [inline CHANGELOG.md:53]:
```suggestion
- [docs] Improve performance on Charts entry point @oliviertassinari
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,7 +3,66 @@
...
+- [docs] Move Heat map to pro (#12170) @alexfauquette
+- [docs] Remove Charts installation next tag call-out (#12133) @LukasTy
+- [docs] cherry pick of #12092 (#12126) @michelengelen
```
Comment (flaviendelangle) [inline CHANGELOG.md:56]:
```suggestion
- [docs] Removed focused prop from demo (#12092) @michelengelen
```

Comment (michelengelen) [reply CHANGELOG.md:56]:
```suggestion
- [docs] Removed `focused` prop from demo (#12092) @michelengelen
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,7 +3,66 @@
...
+
+Same changes as in `@mui/x-date-pickers@6.19.5`.
+
+### Charts / `@mui/x-charts@6.19.5`
+
+- [charts] Allow to skip animation on sparkline bar (#12160) @alexfauquette
+
+### Docs
+
+- [docs] Clarify Pickers `Component composition` section (#12147) @LukasTy
```
Comment (michelengelen) [inline CHANGELOG.md:46]:
```suggestion
- [docs] Clarify Pickers 'Component composition' section (#12147) @LukasTy
```

**Review by michelengelen — APPROVED** [review]
nits only ... :shipit:

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,7 +3,66 @@
...
+- [docs] Fix 301 redirection to StackBlitz @oliviertassinari
+- [docs] Fix 301 to Material UI @oliviertassinari
+- [docs] Fix 301 to Material UI @oliviertassinari
+- [docs] Fix 404 links to translation source @oliviertassinari
+- [docs] Fix dead link to translations @oliviertassinari
+- [docs] Fix typo for AdapterDateFnsV3 (#12037) @flaviendelangle
+- [docs] Improve performance on charts entry point @oliviertassinari
+- [docs] Move Heat map to pro (#12170) @alexfauquette
```
Comment (LukasTy) [inline CHANGELOG.md:54]:
```suggestion
- [docs] Move Heat Map Chart to `pro` plan (#12170) @alexfauquette
```

**Review by LukasTy — APPROVED** [review]
Let's maybe wait for https://github.com/mui/mui-x/pull/12185 🤞 🙈

File: CHANGELOG.md (hunk 5 above)
Comment (alexfauquette) [reply CHANGELOG.md:54]: Sorry, I opened this PR before our discussion on chart naming 😅

```suggestion
- [docs] Move Heatmap to pro (#12170) @alexfauquette
```

**Review by alexfauquette — APPROVED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,7 +3,66 @@
...
+- [charts] Allow to skip animation on sparkline bar (#12160) @alexfauquette
+
+### Docs
+
+- [docs] Clarify Pickers `Component composition` section (#12147) @LukasTy
+- [docs] Fix 301 redirection to StackBlitz @oliviertassinari
+- [docs] Fix 301 to Material UI @oliviertassinari
+- [docs] Fix 301 to Material UI @oliviertassinari
+- [docs] Fix 404 links to translation source @oliviertassinari
+- [docs] Fix dead link to translations @oliviertassinari
```
Comment (alexfauquette) [inline CHANGELOG.md:51]:
```suggestion
- [docs] Fix dead link to translations @oliviertassinari
- [docs] Fix the Treemap illustration (#12189) @danilo-leal
```

A cherry pick waiting to be merged: https://github.com/mui/mui-x/pull/12189

**Pull request merged**